src/main/java/com/google/devtools/build/docgen/templates/attributes/common/target_compatible_with.html

```diff
@@ -0,0 +1,75 @@
+<p>
+<code>List of <a href="../build-ref.html#labels">labels</a>; optional;</code>
+</p>
+
+<p>
+A list of
+<code><a href="platform.html#constraint_value">constraint_values</a></code>
+that must be present in the target platform for this target. This is in
+addition to any constraints already set by the rule type. Constraints are used
+to restrict the list of available target platforms. If the target platform does
+not satisfy all listed constraints then the target is considered "incompatible"
+and will be skipped for building and testing.
```

Comment:
Mention the difference between building the target explicitly vs. part of `:all`?